engine/src/main/resources/org/pentaho/di/trans/steps/pentahoreporting/messages/messages_en_US.properties

```diff
@@ -65,3 +65,6 @@ PentahoReportingOutputMeta.Injection.FIELDNAME=The field name providing the sour
 ReportExportTask.ERROR_0001_TARGET_EXISTS=Target-File exists, but cannot be removed.
 ReportExportTask.USER_EXPORT_COMPLETE=Export complete
 ReportExportTask.USER_EXPORT_FAILED=Export failed
+PentahoReportingOutputDialog.OutputFilenameField.Label=Output file field
+PentahoReportingOutputDialog.InputFilenameField.Label=Report definition file field
+PentahoReportingOutputDialog.UseValuesFromField.Label=Use values from field
```

Comment:
Shouldn't the label be called:
"Accept file name from filed?"
This way it would be equal to S3 File Output and Text File Output which also use this logic.

Reply:
:+1: